django/core/validators.py

```diff
@@ -104,7 +107,101 @@ class URLValidator(RegexValidator):
         r'(?:[/?#][^\s]*)?'  # resource path
         r'\Z', re.IGNORECASE)
     message = _('Enter a valid URL.')
-    schemes = ['http', 'https', 'ftp', 'ftps']
+    schemes = [
```

Comment:
Yes, that's the idea. I'm not sure if we need to include a long list of schemes though. As Claude suggested on the ticket, some minimal validation like alphanumeric + period is probably sufficient. That way we don't have to maintain a list. Any projects that require such a list can supply it.